design/Exercises.md

```diff
@@ -0,0 +1,37 @@
+# Exercises UX Research
+
+## User Story
+When I try exercises to help with my depression I want to have an exercise that I can do that will help as much as possible so I can live a normal life.
+
+## How people solve the problem now
+When I try exercises to help with my depression I run them in clunky antique software so that I can do my exercises and live a normal life.
+- Causality: I'm told it might work by academics, or I have already seen results.
+- Anxiety:
+  - That I won't be able to use the software.
+  - That I won't be able to stick to the exercises for long enough to see results.
+- Motivation: I'm told that completing the exercises will help me live a normal life.
+
+## How we might solve this problem
+When I try exercises to help with my depression I can complete them anywhere, and the app encourages me to keep doing them so that I can stick to them and live a normal life.
+- Causality: Either referral or research.
+- Anxiety:
+  - The exercises are not doable.
+  - I won't be able to stick with it long enough for it to help.
+  - The app might track me.
+- Motivation: I'm told that completing the exercises will help me live a normal life.
+
+## Needs
+- I need encouragement so that I keep using the app past the initial frustration.
+- I need to be able to use the app for extended periods without hurting my eyes.
+- I need the exercises to work on any device and be accessibility friendly.
+- I need to be able to complete the tasks effectively (audio in my own language and other exercise specific concerns).
+- I need to be able to use it without an internet connection.
+
+
+## Workflow
+- Present the exercise, with a button to start, skippable (present if small, first part present if needs to be longer) instructions, and optional settings.
```

Comment:
Can you clarify this? I think I get the general meaning, but I get lost in the specifics.